pkgs/development/tools/ocaml/oasis/default.nix

```diff
@@ -5,7 +5,7 @@ stdenv.mkDerivation {
   name = "ocaml-oasis-0.4.7";
 
   src = fetchurl {
-    url = http://forge.ocamlcore.org/frs/download.php/1635/oasis-0.4.6.tar.gz;
+    url = http://forge.ocamlcore.org/frs/download.php/1635/oasis-0.4.7.tar.gz;
```

Comment:
May I suggest factoring out `version` and using it both in the `name` and `src.url`